CONTRIBUTING.md

```diff
@@ -1,27 +1,43 @@
 # Contributing Guidelines
 
-We love improvements to our tools! Our process for accepting changes has a few steps:
+We love improvements to our tools! There are a few key ways you can help us improve our projects:
 
-1. If you haven't submitted anything before, and you aren't (yet!) a member of our organization, **fork and clone** the repo:
+### Submitting Feedback, Requests, and Bugs
+
+Our process for submitting feedback, feature requests, and reporting bugs usually begins by discussion on [our chat](https://github.com/edgi-govdata-archiving/overview#development) and, after initial clarification, through [GitHub issues](https://help.github.com/articles/about-issues/). Each project repository generally maintains its own set of issues:
+
+        https://github.com/edgi-govdata-archiving/<repository-name>/issues
+
+Some projects have additional templates or sets of questions for each issue, which you will be prompted to fill out when creating one.
+
+Issues that span multiple projects or are about coordinating how we work overall are in the [Overview Issue Tracker](https://github.com/edgi-govdata-archiving/overview/issues).
+
+### Submitting Code and Documentation Changes
+
+We have [project guidelines](./PROJECT.md) for all of the projects hosted in our [GitHub Organization](https://github.com/edgi-govdata-archiving), which new repositories should follow during their creation.
+
+Our process for accepting changes operates by [Pull Request (PR)](https://help.github.com/articles/about-pull-requests/) and has a few steps:
+
+1.  If you haven't submitted anything before, and you aren't (yet!) a member of our organization, **fork and clone** the repo:
 
         $ git clone git@github.com:<your-username>/<repository-name>.git
 
-  Organization members should clone the upsteam repo, instead of working from a personal fork:
+    Organization members should clone the upsteam repo, instead of working from a personal fork:
```

Comment:
yeah the push-as-maintainer is nice and I've used it recently here. But it feels a little less communal. 
I dunno, call me a collectivist or something-- I really dig co-working on branches with people :)